source/_usecases/time-your-function.md

```diff
@@ -0,0 +1,23 @@
+---
+extends: _layouts.usecase
+date: 2018-03-14
+link: https://developer.mozilla.org/en-US/docs/Web/API/Performance/now
+reference: performance.now
+related: interpolate
```

Comment:
Even though interpolation is used here, it's not related "at the use case level"
Can you please this line?

Reply:
Alright got ya!